### PR TITLE
Add license and skeleton code

### PR DESCRIPTION
**Pull request by jsdosa**
Add license and skeleton code for audio framework

### CODE REVIEW

File: framework/src/media/player.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include "player.h"
+
```
Comment (shivgarg) [inline framework/src/media/player.c]: I think you forgot to include media_play

Comment (jsdosa) [reply framework/src/media/player.c]: we don't use media_play.h file

**Comment by Taejun-Kwon** [comment]
Add each file's open api into headers.

**Comment by jsdosa** [comment]
Yes, we will add the necessary headers as we implement each functionality.

File: framework/src/media/player.c (hunk 1 above)
Comment (shivgarg) [reply framework/src/media/player.c]: I mean media_play function

Comment (jsdosa) [reply framework/src/media/player.c]: yes, you right

**Pull request merged**